pkg/operator/controller/controller.go

```diff
@@ -41,157 +30,3 @@ const (
 type Controller interface {
 	Register(context.Context, manager.Manager) error
 }
```

Comment:
I'm wondering why this can't be done in operatorpkg or why we still need it? I'd imagine this whole file gets removed.

Reply:
It's a good question. I'll follow-up with @ellistarn offline to validate why this interface doesn't exist.

Reply:
Closed, we've now added this to operatorpkg: https://github.com/awslabs/operatorpkg/pull/51